libevmasm/GasMeter.cpp

```diff
@@ -258,4 +258,15 @@ unsigned GasMeter::runGas(Instruction _instruction)
 	return 0;
 }
 
-
+u256 GasMeter::dataGas(bytes const& _data, bool _inCreation)
+{
+	if (_inCreation)
```

Comment:
This may look wrong to a newcomer: consider adding a comment explaining that creation data is only sent as tx data and not stored, whereas non-creation code will be sent to storage (and pay the confusingly-named `createDataGas` cost).